src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -250,21 +250,20 @@ export function getStudyIds(groups: Pick<SessionGroupData, 'studies'>[]) {
 export function getSampleIdentifiers(
     groups: Pick<SessionGroupData, 'studies'>[]
 ) {
-    return _.uniqWith(
-        _.flattenDeep<SampleIdentifier>(
-            groups.map(group =>
-                group.studies.map(study => {
-                    const studyId = study.id;
-                    return study.samples.map(sampleId => ({
-                        studyId,
-                        sampleId,
-                    }));
-                })
-            )
-        ),
-        (id1, id2) =>
-            id1.sampleId === id2.sampleId && id1.studyId === id2.studyId
-    );
+    const sampleIds: { [key: string]: SampleIdentifier } = {};
+
+    groups.forEach(group => {
```

Comment:
maybe add a comment explaining how we are obtaining uniqueness

Reply:
Sure. Let me know if the comment on lines 250-255 is what you were looking for. Wasn't sure if you wanted me to explain what constitutes a unique sample, or how the samplieIds data structure enforces uniqueness.